examples/model_selection/plot_multi_metric_evaluation.py

```diff
@@ -0,0 +1,89 @@
+"""Demonstration of multi-metric evaluation on cross_val_score and GridSearchCV
```

Comment:
What do you think about keeping this example and removing [this one instead](http://scikit-learn.org/stable/auto_examples/model_selection/grid_search_digits.html#sphx-glr-auto-examples-model-selection-grid-search-digits-py)?

The `GridSearchCV` documentation has a class example that illustrates a simple single metric evaluation...